compiler/main/driver.cpp

```diff
@@ -270,7 +281,7 @@ static void setupChplHome(const char* argv0) {
   }
 
   if( chpl_home ) {
-    if( strlen(chpl_home) > FILENAME_MAX )
+    if( strlen(chpl_home) >= FILENAME_MAX )
```

Comment:
no, it was right before, since CHPL_HOME has space for FILENAME_MAX + 1. I'll un-do this change.